haystack/pipelines/base.py

```diff
@@ -900,15 +936,16 @@ def __init__(self, address: str = None, **kwargs):
         super().__init__()
 
     @classmethod
-    def load_from_yaml(
+    def _load_from_config(
             cls,
-            path: Path, pipeline_name: Optional[str] = None,
+            pipeline_config: Dict, 
+            pipeline_name: Optional[str] = None,
             overwrite_with_env_variables: bool = True,
             address: Optional[str] = None,
             **kwargs,
     ):
         """
-        Load Pipeline from a YAML file defining the individual components and how they're tied together to form
+        Load Pipeline from a YAML parsed as a dictionary defining the individual components and how they're tied together to form
```

Comment:
doc string below is outdated (example + args)

Reply:
done.